resources/lib/log.py

```diff
@@ -18,6 +19,8 @@
 try:
     import xbmc
     def _log(message, level=_DEFAULT):
+        if level == DEBUG and os.environ.get('DEBUG', 'no') == 'no':
```

Comment:
I think it would be better to store the result of the environment check in a global variable and check that here, this code seems to check environment on each debug log call.

Reply:
I think it would be good to have a `TRACE` log level that does the current behavior, however, I think it's really the `log_function` method that you want to mask (as that's not needed unless developing really)

Reply:
@HiassofT Changed, although I do not see os.environ.get() being an expensive function.

@lrusak Not only log_function(). There are some larger dumps i.e. of keyboard and update files.